config/software/libsodium.rb

```diff
@@ -37,7 +37,11 @@
   source md5: "c224fe3923d1dcfe418c65c8a7246316"
 end
 
-source url: "https://download.libsodium.org/libsodium/releases/libsodium-#{version}.tar.gz"
+if Gem::Version.new("#{version}") > Gem::Version.new("1.0.8")
```

Comment:
[I've just learnt](https://github.com/chef/omnibus-software/commit/9d9a88d79e1d04463a41bf5b3adb382e12b8c1d5#diff-2860526a6e621942b184c12c63d93675R38) there's `.verifies?` on `version`, so this could be
```ruby
if version.statisfies?("> 1.0.8")
```